root/docker/docker-entrypoint

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+USER_NAME=${USER_NAME:-gatling-user}
+echo "User name is set to $USER_NAME"
+
+PUID=${PUID:-911}
+PGID=${PGID:-911}
+
+if [[ "$USER_NAME" != "gatling-user" ]]; then
+  usermod -l "$USER_NAME" gatling-user
+  groupmod -n "$USER_NAME" gatling-user
+fi
+
+groupmod -o -g "$PGID" "$USER_NAME"
+usermod -o -u "$PUID" "$USER_NAME"
+
+echo "
+───────────────────────────────────────
+GID/UID
+───────────────────────────────────────
+User UID:    $(id -u "${USER_NAME}")
+User GID:    $(id -g "${USER_NAME}")
+───────────────────────────────────────
+"
+
+# create folders
+mkdir -p \
+    /config/{.ssh,ssh_host_keys,logs/openssh} \
+    /run/sshd
+
+# symlink out ssh config directory
+if [[ ! -L /etc/ssh ]]; then
+    if [[ ! -f /config/ssh_host_keys/sshd_config ]]; then
+        sed -i '/#PidFile/c\PidFile \/config\/sshd.pid' /etc/ssh/sshd_config
```

Comment:
Why ? We use this pid file ?

Reply:
sshd itself

Reply:
this might be worth more detailed comments 👍

Reply:
To be honest, I got most of the code from [linuxserver](https://github.com/linuxserver/docker-openssh-server/)